### PR TITLE
added information on running tests to readme

### PR DESCRIPTION
**Pull request by DarraghHaughey-Kainos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,11 @@ How to start the DropwizardWebService application
 ---
```
Comment (glenn-hamilton) [inline README.md:13]: Are there any environment variables that are needed to run the application?

**Review by glenn-hamilton — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,13 +1,24 @@
-# I Want It Data Way API
+I Want It Data Way API
+---
+Before running this application, you will need to set up the following environment variables to enable a database connection:
+```
+DB_HOST
+DB_USERNAME
+DB_PASSWORD
+DB_NAME
+```
```
Comment (steveh2707) [inline README.md:10]: Add JWT_SECRET environment variable

File: README.md [hunk 3]
```diff
@@ -1,13 +1,24 @@
...
 
-Health Check
+Swagger
 ---
+To see your applications Swagger UI `http://localhost:8080/swagger`
 
-To see your applications health enter url `http://localhost:8081/healthcheck`
+Tests
+---
+1. Run `mvn clean test` to run unit tests.
```
Comment (steveh2707) [inline README.md]: remove unit

**Review by steveh2707 — APPROVED** [review]
Looks good with minor changes

**Pull request merged**